smithy-openapi/src/test/java/software/amazon/smithy/openapi/fromsmithy/OpenApiJsonSchemaMapperTest.java

```diff
@@ -67,23 +66,6 @@ public void convertsModels() {
         assertTrue(document.toNode().expectObjectNode().getMember("components").isPresent());
     }
 
-    @Test
-    public void stripsUnsupportedKeywords() {
```

Comment:
This behavior is now covered by the `convertsToOpenAPI3_0_2` and `convertsToOpenAPI3_1_0` tests in OpenApiConverterTest.